.history/src/google_takeout_metadata/sidecar_20250906083321.py

```
@@ -0,0 +1,225 @@
+        filename=title,
+        description=description,
+        people=people,
+        taken_at=taken_at,
+        created_at=created_at,
+        latitude=latitude,
+        longitude=longitude,
+        altitude=altitude,
+        favorite=favorite,
+        lat_span=lat_span,
+        lon_span=lon_span,
+        image_views=image_views,
```

Ruff (0.12.2)

Undefined name `image_views`

(F821)